.github/workflows/dockerhub-master.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: build-tag-push-master
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
```

Comment:
As per GitHub Action's warnings: we shouldn't rely on `master` branch for `actions/checkout` because `master` is not the default branch for that repo anymore. So I'm specifically using `v2` now.